src/components/KDropdown/KDropdownItem.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+  <li
+    class="k-dropdown-item"
+    :class="{
+      'has-divider': hasDivider,
+      'disabled': disabled,
+      'danger': isDangerous,
+      'k-dropdown-selected-option': selected
+    }"
+    :data-testid="`k-dropdown-item-${label.replace(/ /gi, '-')}`"
+  >
+    <component
+      :is="availableComponents[componentType].tag"
+      data-testid="k-dropdown-item-trigger"
```

Comment:
question: should this testid change like the others?

Reply:
Yes, done now.